### ISSUE TITLE
Automatic tensor network visualization

### ISSUE THREAD

**Issue by evaleev** [ValeevGroup/SeQuant#21]

**Comment by Krzmbrzl** [comment]
Don't we have automatic visualization by means of being able to produce dot code? :thinking:

**Comment by bimalgaudel** [comment]
Dot graph can be generated for the binarized expressions. TensorNetwork visualization here implies more general graph generation rather than converting the graph into a tree as happens in the binarization.

**Comment by evaleev** [comment]
@Krzmbrzl chemists/physicists expect particular forms of graphical representations of tensor networks in the case of many-body methods ("diagrams"), and particular layout rules for tensor networks. Generic dot will produce something somewhat usable, but without layout constraints the output will not be what most expect.

**Comment by Krzmbrzl** [comment]
Yeah, I guess that makes sense. In this context, maybe https://doi.org/10.1007/s00214-019-2505-3 could be of interest.
